.github/workflows/update_online.yml

```diff
@@ -0,0 +1,29 @@
+name: Update the online lists
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '16 20 * * 5'
+  workflow_dispatch:
+jobs:
+  mkonline:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Build
+      run: |
+        cd scripts
+        python update_online.py
+    - name: Commit & Push
+      uses: actions-x/commit@v2
+      with:
+          email: noreply@example.com
+          name: Update bot
+          branch: master
```

Comment:
This is triggered on `pull_request` but would push to the master branch. That will lead to a situation where users can submit pull requests and modify master without any approval.

Reply:
I did not realize that would be possible.
Let me correct it

Reply:
Corrected